### PR TITLE
Feature: Customer Address CRUD

### PR DESCRIPTION
**Pull request by wpiyaphon**
CRUD for customer address

### CODE REVIEW

File: src/app/api/customer-address/route.ts
Comment (tarlrsk) [inline src/app/api/customer-address/route.ts]: 404

Comment (panlabdev) [inline src/app/api/customer-address/route.ts]: No need to check for api method

File: src/app/api/customer-address/[id]/route.ts
Comment (panlabdev) [inline src/app/api/customer-address/[id]/route.ts]: Try it after authentication is merged

**Review by panlabdev — CHANGES_REQUESTED** [review]
Check HTTP status code documentation for more info. 404 is for Not Found not 400.

**Comment by panlabdev** [comment]
@wpiyaphon This should be good. After the postman is done DM me. We can merge this once it is done.

**Review by panlabdev — APPROVED** [review]
LGTM

**Comment by panlabdev** [comment]
@tarlrsk Fix vercel please

**Review by panlabdev — APPROVED** [review]

**Pull request merged**
